api/envoy/admin/v3/config_dump.proto

```diff
@@ -381,3 +381,18 @@ message EndpointsConfigDump {
   // The dynamically loaded endpoint configs.
   repeated DynamicEndpointConfig dynamic_endpoint_configs = 3;
 }
+
+// Message of unready targets information of an init manager.
```

Comment:
Could you add an example of the typical name and typical target_name?
Much easier for the doc readers.

Reply:
Sure, will add "init_manager_xxx" and "target_xxx"